src/LogOutputManager.ts

```diff
@@ -145,11 +154,37 @@ export class LogOutputManager {
     /**
      * Log output methods
      */
-
     public appendLine(lineText: string, mustInclude: boolean = false): void {
         let lines = lineText.split('\n');
         lines.forEach((line) => {
             if (line !== '') {
+                if (!(this.config.output || {}).includeStackTraces) {
+                        // filter out debugger noise
+                    if (line.match(this.debugStartRegex)) {
+                        console.log('start MicroDebugger block');
+                        this.isInMicroDebugger = true;
+                        this.currentMicroDebuggerText = '';
+                        this.isNextBreakpointSkipped = false;
+                        line = 'Pausing for a breakpoint...';
+                    } else if (this.isInMicroDebugger && line.match(this.debugEndRegex)) {
+                        console.log('ended MicroDebugger block');
+                        this.isInMicroDebugger = false;
+                        if (this.isNextBreakpointSkipped) {
+                            line = '\n**Was a bogus breakpoint** Skipping!\n';
+                        } else {
+                            line = null;
+                        }
+                    } else if (this.isInMicroDebugger) {
+                        this.currentMicroDebuggerText += line + '\n';
+                        if ((this.config.debug || {}).skipBogusBreakpoints && line.startsWith('Break in ')) {
```

Comment:
This line assumes the user set `skipBogusBreakpoints` in the user/workspace settings. However, a perfectly valid way of setting `skipBogusBreakpoints` is in the launch config (_without_ using settings at all)  meaning this code path won't always work with `skipBogusBreakpoints`. 

In [DebugConfigurationProvider around line 139](https://github.com/TwitchBronBron/vscode-brightscript-language/blob/master/src/DebugConfigurationProvider.ts#L102), we have a pattern of storing a value in workspaceState. I think this is a better approach. 

So, at the end of the debug config provider, run `this.context.workspaceState.update('skipBogusBreakpoints, config.skipBogusBreakpoints)`, and then in the code above, read that value ` await this.context.workspaceState.get('skipBogusBreakpoints')` instead of getting it from settings.